app/composer.json

```diff
@@ -106,8 +105,11 @@
     "composer/package-versions-deprecated": "1.11.99.1",
     "phpoffice/phpspreadsheet": "^1.15",
     "predis/predis": "^1.1",
-    "composer/composer": "^2.2",
-    "beberlei/doctrineextensions": "^1.3"
+    "symfony/amazon-mailer": "^5.4",
```

Comment:
Shouldn't this (and the MailerSesBundle) be part of an extra plugin and not the core dependency as described in the description?